scripts/base/frameworks/storage/main.zeek

```diff
@@ -10,6 +10,11 @@ export {
 	type BackendOptions: record {
 		## The serializer used for converting Zeek data.
 		serializer: Storage::Serializer &default=Storage::STORAGE_SERIALIZER_JSON;
+
+		## Sets the backend into forced-synchronous mode. All operations will run
+		## in synchronous mode, even if the async functions are called.  This
+		## should generally only be set to ``T`` during testing.
+		forced_synchronous : bool &default=F;
```

Comment:
```suggestion
		forced_sync : bool &default=default_forced_sync;
```

Mostly so the naming is "in sync" with the `Storage::Sync` module and the `is_forced_sync()` bif and also a bit separate from the "synchronous" pragma that SQLite has which is quite unrelated. The default to allow switching all backends used in any script into `forced_sync` mode by doing `redef Storage::default_forced_sync = T`, without needing to change each individual script that opens a storage backend. Think that's something @J-Gras  would do.

The latter makes me actually wonder if it should even be part of `BackendOptions`, but I guess that provides some flexibility.